cmd/account/get_test.go

```diff
@@ -146,14 +160,26 @@ const malformedAccountGetJson = `
 }
 `
 
+//TODO(karlkfi): Why is the json consistently ordered but not alphabetical?
```

Comment:
Generally I'd write a test like this by parsing the output JSON as then asserting properties about it, e.g. (pseudocode) `assert json.get("name") == "account"` for all 7 properties, and then `assert json.properties().count() == 7`. That insulates you from changes in the exact formatting of the output, i.e. if you upgrade a library and the sorting changes or it starts adding quotes around the YAML values or something...

But that would be a massive change and I'm not trying to imply you should do that, just FYI...

Reply:
There's no real ordering validation happening elsewhere, due to the general lack of UI and output tests... So I was trying to make sure there was at least ordering test coverage somewhere...

I have no idea why the API client mutates the order received from the server tho... when I make this alphabetical as what `testGateAccountGetSuccess()` returns, the test fails because the return from the `spin account get` command is sorted in some consistent, non-alphabetical way. I'm pretty sure this is a client bug, but I haven't found it...

Reply:
So I found the weird consistent ordering. It's in the swagger API definition:
https://github.com/spinnaker/spin/blob/master/gateapi/api/swagger.yaml#L6055-L6088

And it gets propagated to the AccountDetails go object:
https://github.com/spinnaker/spin/blob/master/gateapi/account_details.go#L12

And the client returns an AccountDetails object instead of a map[string]interface{}:
https://github.com/spinnaker/spin/blob/master/gateapi/credentials_controller_api.go#L48

So the JsonOutput formats it in the order from the Object... crazy.